### PR TITLE
Adds admin away gateway.

### PR DESCRIPTION
**Pull request by AnturK**
Because setting this up by hand is bit annoying.

### CODE REVIEW

File: code/modules/awaymissions/gateway.dm [hunk 1]
```diff
@@ -233,6 +233,17 @@ GLOBAL_DATUM(the_gateway, /obj/machinery/gateway/centerstation)
 			M.client.move_delay = max(world.time + 5, M.client.move_delay)
 
 
+/obj/machinery/gateway/centeraway/admin
```
Comment (ninjanomnom) [inline code/modules/awaymissions/gateway.dm]: This can just be removed, overriding the proc below makes this a subtype

**Review by ninjanomnom — DISMISSED** [review]

File: code/modules/awaymissions/gateway.dm (hunk 1 above)
Comment (lzimann) [reply code/modules/awaymissions/gateway.dm]: An empty declaration is better than going to the proc right away imo

**Comment by Armhulen** [comment]
so like what is it

**Comment by AnturK** [comment]
One tile gateway that auto-links with station one, aka spawn where you want the gateway to go and forget.

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**